src/main/kotlin/no/nav/tilleggsstonader/sak/vedtak/OpphørValideringService.kt

```diff
@@ -89,34 +90,34 @@ class OpphørValideringService(
         }
     }
 
-    private fun validerIngenEndredePerioderMedTomEtterRevurderFraDato(
+    private fun validerIngenEndredePerioderMedTomEtterOpphørsdato(
```

Comment:
Burde denne valideringen bare byttes ut med en sjekk om opphørsdato >= dato for tidligste endring